package/exec-logrotate

```diff
@@ -17,6 +17,14 @@ function create_logrotate_config() {
     notifempty
     copytruncate
 }
+
+/log/prestop-${DATA_ENGINE}.log {
+    rotate 2
+    size 20M
+    missingok
+    notifempty
+    copytruncate
+}
```

Comment:
Hmm... Do we need logrotate for the cleanup script? I am thinking it's a one-time job.

Reply:
We append the prestop messages to the log file, so it is better to rotate.

Reply:
I mean, do we need to do rotation? Anyway, this does not hurt anything. Let me approve it first